### PR TITLE
PEP572 Use Python's assignment expression

### PR DESCRIPTION
**Pull request by cclauss**
As suggested by @oddbookworm at https://github.com/pygame-community/pygame-ce/pull/2770#pullrequestreview-1962131646
* https://peps.python.org/pep-0572

### CODE REVIEW

**Review by oddbookworm — CHANGES_REQUESTED** [review]
Just one small change I'd like to see, but overall looks good!

File: test/mixer_test.py [hunk 1]
```diff
@@ -384,8 +383,7 @@ def _test_array_argument(self, format, a, test_pass):
             )
         a2 = array(snd)
         a3 = a.astype(a2.dtype)
-        lshift = abs(format) - 8 * a.itemsize
-        if lshift >= 0:
+        if (lshift := abs(format) - 8 * a.itemsize) >= 0:
```
[the remark this answers is not in the document]
Comment (cclauss) [reply test/mixer_test.py:386]:
```suggestion
        lshift = abs(format) - 8 * a.itemsize
        if lshift >= 0:
```

**Comment by cclauss** [comment]
Failing test is fixed in
* #2783

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for the PR :tada:

**Pull request merged**